chia/_tests/core/test_full_node_rpc.py

```diff
@@ -150,8 +150,9 @@ async def test1(
 
         assert len(await client.get_all_mempool_items()) == 0
         assert len(await client.get_all_mempool_tx_ids()) == 0
-        assert (await client.get_mempool_item_by_tx_id(spend_bundle.name())) is None
-        assert (await client.get_mempool_item_by_tx_id(spend_bundle.name(), False)) is None
+        with pytest.raises(ValueError, match="not in the mempool"):
+            await client.get_mempool_item_by_tx_id(spend_bundle.name())
+            await client.get_mempool_item_by_tx_id(spend_bundle.name(), False)
```

Comment:
I'm pretty sure this line won't be executed. I think it would need its own `with pytest.raises(...):`  context